module_analysis/readme/CONFIGURE.rst

```diff
@@ -16,9 +16,9 @@ developped for exemple,
 
 to update the data manually, you have to :
 
-* Go to 'Apps' / 'Update Apps List'
+* Go to 'Settings' / 'Technical' / 'Scheduled Actions'
 
-* Check the box 'Analyse Installed modules'
+* Run manually the action : 'Update Module Analysis'
 
     .. image:: ../static/description/base_module_update.png
```

Comment:
I guess that line has to be removed. (and the according png file).

Reply:
I've added the line => To run an update manually, you can run the scheduled action manually.
The image is indeed to remove